src/AppHosts/SuperMemoAssistant/CollectionSelectionWindow.xaml.cs

```diff
@@ -244,6 +246,36 @@ private void ListBoxItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
       lbCollections.SelectFirstItem();
     }
 
+    private void ShowQuoteOfTheDay(object sender, RoutedEventArgs e)
+    {
+
+      var QuoteFile = SMAFileSystem.AppRootDir.CombineFile("quotes.csv");
+
+      if (QuoteFile.Exists())
+      {
+        var Lines = File.ReadAllLines(QuoteFile.FullPath);
```

Comment:
There are several scenarios where `File.ReadAllLines` could throw an exception, can you please:
- handle `FileIOException` with a `LogTo.Warning`,
- handle `Exception` (fallback) with a `LogTo.Error` (this will file a bug report as well as logging)
- let the try... catch encompass the whole if code branch (under the if), for safety
- gracefully continue the code if an exception is thrown (don't rethrow or exit)